lib/razorpay/item.rb

```diff
@@ -0,0 +1,34 @@
+require 'razorpay/request'
+require 'razorpay/entity'
+
+module Razorpay
+  # Item API allows you to create and fetch customers on Razorpay
+  class Item < Entity
+    def self.request
+      Razorpay::Request.new('items')
+    end
+
+    def self.create(options)
+      request.create options
+    end
+
+    def self.fetch(id)
+      request.fetch id
+    end
+
+    def self.edit(id, options = {})
+      if(!options.is_a?(String) && options.key?(:active))
+        options[:active] = (options[:active] ? 1 : 0)
+      end 
+      request.patch id, options
+    end
+
+    def self.all(options = {})
+      request.all options
+    end
```

Comment:
Based on docs, https://razorpay.com/docs/api/items/#fetch-multiple-items, what options are we expecting?

Reply:
its working with parameter like `from, to, count, active` like other `all` api

Reply:
Pls confirm from API owner